zips/zip-0317.rst

```diff
@@ -8,7 +8,11 @@
            Kris Nuttycombe
            Jack Grigg
            Francisco Gindre
-  Status: Active
+           Daniel Benarroch
+           Jonathan Rouach
+           Pablo Kogan
+           Vivek Arte
+  Status: Draft
```

Comment:
```suggestion
  Status: [Revision 0] Active, [Revision 1] Draft
```
See e.g. ZIP 214 for an existing use of this convention for NU6 changes.

Reply:
I updated it like so based on how it was in ZIP 316 (though now I see that has been updated to Proposed)
```
Status: Revision 0: Active, Revision 1: Draft
```